src/game/objects/player.ts

```diff
@@ -7,6 +7,12 @@ export type Directions = {
     right: boolean
 }
 
+const baseSpeed = 13
+const itemEventBlock = 6
+const teleportDistanse = 30
+
+export { baseSpeed, itemEventBlock, teleportDistanse }
```

Comment:
```suggestion
// player.constant.ts

const BASE_SPEED = 13
const ITEM_EVENT_BLOCK = 6
const TELEPORT_DISTANCE = 30

const CHARACTER =  { BASE_SPEED, ITEM_EVENT_BLOCK, TELEPORT_DISTANCE }

export default CHARACTER

// CHARACTER.BASE_SPEED
```



상수로 뺀거 너무 좋습니다! 다만, 다음과 같은 컨벤션을 제안드려요!
1. 상수가 많아질 것을 대비해서 player.constant.ts 파일로 분리하면 어떨까요? 그리고 캐릭터에서 사용되는 상수라는걸 인지하기 위해 객체로 감싸면 좋을 것 같습니다. (추후 아이템 상수와 분리하기 위함)
2. 그리고 상수 같은 경우 대문자로 별칭을 두면, 코드 가독성이 높아질 것 같아요! [참고](https://ko.javascript.info/variables#ref-786)